packages/flutter_tools/test/integration/flutter_tester_test.dart

```diff
@@ -93,3 +130,16 @@ dependencies:
     sdk: flutter
 ''');
 }
+
+Future<void> _getPackages() async {
+  final List<String> command = <String>[
+    fs.path.join(getFlutterRoot(), 'bin', 'flutter'),
+    'packages',
+    'get'
+  ];
+  final Process process = await processManager.start(command);
+  process.stderr.transform(utf8.decoder).listen(print);
```

Comment:
Does this print to the console during running tests?
We probably don't want this.

Reply:
Whoops! I've changed this to a `StringBuffer` and include it in the exception text if there's a non-zero exit code now.